PyRDF/backend/Local.py

```diff
@@ -13,6 +13,26 @@ class Local(Backend):
         backend.
 
     """
+    def __init__(self, config={}):
+        """
+        Creates a new instance of the
+        Local implementation of `Backend`.
+
+        Parameters
+        ----------
+        config (optional)
+            The config object for the required
+            backend. The default value is an
+            empty Python dictionary `{}`.
+
+        """
+        super().__init__(config)
+        operations_not_supported = [
+        'Take',
+        'Snapshot',
+        'Foreach'
+        ]
+        self.supported_operations = [op for op in self.supported_operations if op not in operations_not_supported]
```

Comment:
```python
if not ROOT.ROOT.IsImplicitMTEnabled():
  self.supported_operations.append('Range')
```

Reply:
@etejedor , can I give this as a part of the multi threading PR ?

Reply:
And also, I can use `config` instead ROOT itself right ?

Reply:
I suggest you use for now the IsImplicitEnabled method, since we haven't yet introduced the config options for the local backend. Anyway, you can always rely on that method, even if you check the config option before and activate the implicit multi-threading yourself in the Local constructor.

Let's just include this line in this PR already, it does not hurt.

Reply:
Ok @etejedor :+1:

Reply:
There's one more issue with this @etejedor . What if the user enables multi-threading after the creation of this constructor ?

Reply:
So, in that case, I think I must add the check to `check_supported` itself then

Reply:
I've made that change anyways @etejedor 👍

Reply:
Yes it's a bit tricky, even adding a check in `check_supported` would not completely solve the issue, because the user might activate this flag after all the nodes of the graph have been created and right before the actual execution (access to a Proxy).
I think for now we can just assume that IMT will be activated already before the Local backend is constructed.

Reply:
Yes @etejedor . I agree. We can look into more cases in the multi-threading PR, which is coming up next 👍